apps/mobile/app/services/hooks/useTeamInvitation.ts

```diff
@@ -68,6 +72,14 @@ export function useTeamInvitations() {
 		).then((res) => console.log(res))
 	}, [])
 
+	const removeSentInvitation = useCallback(async (inviteId: string) => {
+		await removeTeamInvitationsRequest({
```

Comment:
@desperado1802 can we remove the card at the same time user remove the invite, updating local state and then do API call to remove?

We want to make the app very responsive. Even we will be doing that in web too gradually.